### ISSUE TITLE
MERRA `prcp` doesn't seem to be `mm d-1`

### ISSUE THREAD

**Issue by ircwaves** [Applied-GeoSolutions/gips#228]
Skipping to what makes me say that:
```python
>>> i = gippy.GeoImage('/data2/merra/tiles/h01v01/2016029/h01v01_2016029_merra_prcp.tif') 
>>> a = i.Read() 
>>> a.max() 
3993.8394 
```
and qgis computes the 98th percentile as 200, which still seems like too many `mm` for one day.  What say you @bhbraswell.

**Comment by bhbraswell** [comment]
I don't think the max is best test. The data are global and include some pretty weird places that might get 3 m of precipitation in a day. I'll look into it. Note there are 24 layers (1 for each hour) so you have to account for that.

```
            elif val[0] == "prcp":
                # TODO: check this units conversion from (kg m-2 s-1) to (mm d-1)
                fun = lambda x: x.mean(axis=0)*36.*24.*1000.
                units = "mm d-1"
                self.write_reduced(val[0], fun, fout, meta, units)
```

**Comment by ircwaves** [comment]
Totally agree that max is not awesome, but clicking around that TIF, the high-precip areas seem to run in the neighborhood of 300.

**Comment by ircwaves** [comment]
Out of curiosity, (if I read that `fun` right) why the 24-hour `mean` precip value, instead of just `sum`ming the layers?  dropping outliers?

**Comment by bhbraswell** [comment]
So this conversion. We are going from 

```kg m-2 s-1``` to ```mm d-1```


I've done it many times and I always got the same answer, until now. I believe that is in the driver is off by a factor of 10. I think it should be

```[mm d-1] = 3600*24*[kg m-2 s-1]```

Could someone ... @ircwaves @justinfisk ... anyone do this and confirm or deny my result?

**Comment by bhbraswell** [comment]
OK, I did it three more times and got the same answer.

**Comment by bhbraswell** [comment]
When this is merged we'll need to remove or reprocess all the merra prcp product holdings.

**Comment by ircwaves** [comment]
I was typing my "looks good to me" response last night when the network failed. 

@justinfisk -- maybe you close this issue once the reproc is off and running?

**Comment by ircwaves** [comment]
fixed 17cfe2a
